.github/workflows/changelog.yml

```diff
@@ -16,14 +16,13 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-        with:
-          # Only the latest commit of the feature branch is available
-          # automatically. To diff with the base branch, we need to
-          # fetch that too (and we only need its latest commit).
-          fetch-depth: 1
```

Comment:
According to the documentation, `fetch-depth: 1` is already the default, so, this is a noop. https://github.com/actions/checkout#usage